src/v/raft/heartbeat_manager.cc

```diff
@@ -244,7 +244,7 @@ ss::future<> heartbeat_manager::do_heartbeat(node_heartbeat&& r) {
                  r.target,
```

Comment:
Right, a plain "duration" like 500 ms (not anchored to any time) often makes sense to pass to an API which is going to do the timeout internally, since they will be responsible for the now() call. In our existing RPC API, however, we let the caller do the calculation, i.e., we take a time point already anchored in time. This has some advantages, like letting callers enforce an overall timeout when they do some work up front before calling into the RPC layer. 

So I didn't really change this: the RPC send API still takes what amounts to a moment in time (the timeout point), but now includes a duration so it's really a range of time, form "start" to "timeout". Other way of thinking about it is two points in time.

I do have a change which I don't want to try to backport which clears this up a lot more tree-wide, changing all callers to use only `timeout_spec` and not the other overloads, which will be removed.